Casks/c/clamxav.rb

```diff
@@ -40,17 +40,10 @@
 
   zap trash: [
     "/Library/Application Support/ClamXAV",
-    "~/Library/Caches/uk.co.canimaansoftware.ClamXAV",
-    "~/Library/Caches/uk.co.canimaansoftware.ClamXAV-UI-Helper",
-    "~/Library/Caches/uk.co.canimaansoftware.ClamXAV.XAV",
-    "~/Library/HTTPStorages/uk.co.canimaansoftware.ClamXAV",
-    "~/Library/HTTPStorages/uk.co.canimaansoftware.ClamXAV-UI-Helper",
-    "~/Library/HTTPStorages/uk.co.canimaansoftware.ClamXAV.binarycookies",
-    "~/Library/HTTPStorages/uk.co.canimaansoftware.ClamXAV.XAV",
-    "~/Library/Logs/ClamXAV-Console.log",
-    "~/Library/Logs/ClamXAV-UI-Helper-Console.log",
-    "~/Library/Preferences/uk.co.canimaansoftware.ClamXAV-UI-Helper.plist",
-    "~/Library/Preferences/uk.co.canimaansoftware.ClamXAV.plist",
+    "~/Library/Caches/uk.co.canimaansoftware.ClamXAV*",
+    "~/Library/HTTPStorages/uk.co.canimaansoftware.ClamXAV*",
+    "~/Library/Logs/ClamXAV-*Console.log",
```

Comment:
```suggestion
    "~/Library/Logs/ClamXAV*-Console.log",
```

A small nit since `-Console` is the suffix.